src/app/members/actions/delete-user.test.ts

```diff
@@ -0,0 +1,13 @@
+import { waitFor } from "@testing-library/react";
+import { describe, expect, it, vi } from "vitest";
+import { deleteMember } from "./delete-user";
+
+describe("Test Delete User Function", () => {
+  it("Delete Function should be called", () => {
```

Comment:
Tests for who can delete member. Also, if I'm not wrong, it's not delete, it's deactivate. cc @muhammad-bilal-ali-khokhar